Samples/hello-world/outlook-hello-world/README.md

```diff
@@ -94,6 +99,36 @@ function sayHello() {
 
 For more information see [Build your first Outlook add-in](https://learn.microsoft.com/office/dev/add-ins/quickstarts/outlook-quickstart)
 
+## Run the Teams manifest version of hello world
+
+If you want to run the hello world sample using the preview Teams manifest (**manifest.json** or **manifest-localhost.json**) follow the steps below. The Teams manifest files perform the exact same function as the XML manifest files. They help Outlook get information about your add-in, and load the correct resources. For more information, see [Teams manifest for Office Add-ins (preview)](https://learn.microsoft.com/office/dev/add-ins/develop/json-manifest-overview).
+
+>Note: The new manifest is available for preview and is subject to change based on feedback. We encourage experienced add-in developers to experiment with it. The preview manifest should not be used in production add-ins.
+
+1. Follow the instructions at [Set up your development environment](https://learn.microsoft.com/office/dev/add-ins/overview/set-up-your-dev-environment?tabs=yeomangenerator) to ensure you have the required development environment with Node.js and Visual Studio Code.
+
+1. From the root folder of this sample, run the following command to install the **office-addin-debugging** package. This will also create a package.json file.
```

Comment:
Prior to this, consider adding a step to fork and clone the sample repo or to download the sample folder.

Reply:
added, thanks!